src/Raven.Studio/wwwroot/Content/css/_bs5.scss

```diff
@@ -52,6 +52,7 @@
 
     // Pages
     @import "../scss/pages/stats.scss";
+    @import "../scss/pages/splashscreen.scss";
```

Comment:
this should be referenced from component instead of global import. Please talk with @kwiato about that. :)

Reply:
Ok, I've changed that